### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: doc/administrator/administrate.rst.mako [hunk 1]
```diff
@@ -105,7 +105,7 @@ On WMTS layers we have the following specific attributes:
 * ``Style``: the used style, if not present we use the default style.
 * ``Matrix set``: the used matrix set, if there's only one matrix set
   in the capabilities it can be empty.
-* ``WMTS Dimensions``: the dimensions, if not provided default values are used.
+* ``Dimensions``: the WMTS dimensions, if not provided default values are used.
```
Comment (sbrunner) [inline doc/administrator/administrate.rst.mako]: Shouldn't be moved int the `All layer type` chapter (without WMTS)?

Comment (gberaudo) [inline doc/administrator/administrate.rst.mako]: Done.

**Comment by sbrunner** [comment]
This merged changes is wrong !!!

**Comment by sbrunner** [comment]
You don't see my comments?

**Comment by gberaudo** [comment]
Now I can see them, see the follow up PR #2581.

**Pull request merged**